src/zc/zodbrecipes/tests.py

```diff
@@ -15,15 +15,10 @@
 from zope.testing import renormalizing
 
 import doctest
-import os
-import pkg_resources
 import re
-import shutil
-import sys
-import tempfile
 import unittest
 import zc.buildout.testing
-import zope.testing
+import zope.testing  # noqa: F401
```

Comment:
Why do we need this import here?

Reply:
For me it looks like it gets used in line 49
```
    zc.buildout.testing.install('zope.testing', test)
```
but who knows? This doctest thingy is still a mysery.

Reply:
No, that line easy_installs zope.testing into the temporary buildout working tree.

I think the unused zope.testing import can be safely removed.

Reply:
Thank you!